glide.yaml

```diff
@@ -1,9 +1,5 @@
 package: github.com/sorintlab/stolon
 import:
-- package: github.com/coreos/rkt
-  version: v1.16.0
-  subpackages:
-  - pkg/lock
```

Comment:
you should also remove the directory inside vendor: `vendor/github.com/coreos/rkt`